packages/mui-lab/src/DesktopDateRangePicker/DesktopDateRangePicker.tsx

```diff
@@ -452,6 +456,11 @@ DesktopDateRangePicker.propTypes /* remove-proptypes */ = {
    * @default false
    */
   showDaysOutsideCurrentMonth: PropTypes.bool,
+  /**
+   * If `true`, the today button is displayed. **Note** that `showClearButton` has a higher priority.
```

Comment:
What does that mean? A test case should be added for this.

Reply:
I guess this should not be present in DateRangePicker, because showTodayButton would not be any useful while selecting a range. What do you say?

Reply:
I would expect that it sets the current range limit to today. So first you get the picker for start, click "today" and then the picker for end and select another date.

But that's not really my main point. My original point would apply to TimePicker, DatePicker etc.